pkg/action/install.go

```diff
@@ -785,8 +785,18 @@ func (c *ChartPathOptions) LocateChart(name string, settings *cli.EnvSettings) (
 		dl.Verify = downloader.VerifyAlways
 	}
 	if c.RepoURL != "" {
-		chartURL, err := repo.FindChartInAuthAndTLSAndPassRepoURL(c.RepoURL, c.Username, c.Password, name, version,
-			c.CertFile, c.KeyFile, c.CaFile, c.InsecureSkipTLSverify, c.PassCredentialsAll, getter.All(settings))
+		chartURL, err := repo.FindChartInRepoURL(
+			c.RepoURL,
+			name,
+			version,
+			c.CertFile,
+			c.KeyFile,
+			c.CaFile,
+			getter.All(settings),
+			repo.WithUsernamePassword(c.Username, c.Password),
+			repo.WithInsecureSkipTLSverify(c.InsecureSkipTLSverify),
+			repo.WithPassCredentialsAll(c.PassCredentialsAll),
```

Comment:
If options with `.With*` functions are being used for some things, why are they not used for others? I'm thinking about consistency. I like the use of options here.

See https://github.com/helm/helm/blob/0d66425d9a745d8a289b1a5ebb6ccc744436da95/pkg/getter/getter.go#L58 for why I'm thinking about this.

Reply:
Loosely, required arguments IMHO should be named. Optional things should be options. 

That said, good spot here. I initially just converted the optional arguments from the function "overloads". but digging further, the client TLS and chart version are also optional.